h2o-py/h2o/estimators/glm.py

```diff
@@ -416,6 +426,41 @@ def __init__(self,
                binning.
                Defaults to ``-1``.
         :type gainslift_bins: int
+        :param linear_constraints: Linear constraints: used to specify linear constraints involving more than one
+               coefficients in standard form.  It is only supported for solver IRLSM.  It contains four columns: names
+               (strings for coefficient names or constant), values, types ( strings of 'Equal' or 'LessThanEqual'),
+               constraint_numbers (0 for first linear constraint, 1 for second linear constraint, ...
+               Defaults to ``None``.
+        :type linear_constraints: Union[None, str, H2OFrame], optional
+        :param init_optimal_glm: If true, will initialize coefficients with values derived from GLM runs without linear
+               constraints.  Only available for linear constraints.  Default to false.
+               Defaults to ``False``.
+        :type init_optimal_glm: bool
+        :param separate_linear_beta: If true, will keep the beta constraints and linear constraints separate.  After new
+               coefficients arefound, first beta constraints will be applied followed by the application of linear
+               constraints.  Note that the beta constraints in this case will not be part of the objective function.  If
+               false, will combine the beta and linear constraints.  Default to false.
+               Defaults to ``False``.
+        :type separate_linear_beta: bool
+        :param constraint_eta0: For constrained GLM only.  It affects the setting of eta_k+1=eta_0/power(ck+1, alpha).
+               Default to 0.1258925.
+               Defaults to ``0.1258925``.
```

Comment:
The generator generates the information about the default value so it ends up in there twice.